### ISSUE TITLE
more accuracy score

### ISSUE THREAD

**Issue by kushal121231** [DARK-art108/Bank-Note-Authentication-End-to-End-Project-1#1]
we can increase the score to 0.9951456310679612 by using the random state=42 in train_test_split with test size as 0.3 
@DARK-art108
